### PR TITLE
294 Add Getting Started vignette, update README

### PR DESCRIPTION
**Pull request by edelarua**
Closes https://github.com/insightsengineering/coredev-tasks/issues/294

### CODE REVIEW

**Comment by pawelru** [comment]
Please note that vignette needs to be named as the package in order to have it as a "Get Started" article - see: https://pkgdown.r-lib.org/reference/build_site.html#:~:text=%22Get%20Started%22%2C%20which%20links%20to%20a%20vignette%20with%20the%20same%20name%20as%20the%20package.

File: vignettes/nestcolor.Rmd [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+The following NEST packages are optimized to implement `nestcolor`:
+
+- `tern`
+- `chevron`
+- `goshawk`
+- `osprey`
+- `teal.modules.clinical`
+- `teal.modules.general`
+- `teal.goshawk`
+- `teal.osprey`
```
Comment (pawelru) [inline vignettes/nestcolor.Rmd:29]: Please do not couple hard-coded text with the implementation somewhere else. This section is likely to be outdated in the future. What if we would implement nestcolor in let's say teal.modules.hermes? Do we need to update this vignette then? 

I propose to skip it or make this statement more general. Would be safer and easier to maintain.

Comment (edelarua) [reply vignettes/nestcolor.Rmd]: I have removed this section.

**Review by shajoezhu — APPROVED** [review]
Very nice!!! Thanks @edelarua

**Pull request merged**